_test.py

```diff
@@ -18,6 +18,7 @@
   parser = argparse.ArgumentParser()
   parser.add_argument("-clean", help="clean run, as if run for the first time", action="store_true")
   parser.add_argument("-single_threaded", help="run tests in single threaded mode", action="store_true")
+  parser.add_argument("-filter_lines", help="filter lines to only display warnings and errors", action="store_true")
```

Comment:
Kind of a weird name for this argument don't you think?
I had to read the description to see what it does.